### PR TITLE
ISS-244 - Dependency clean up for JUnit Jupiter Zerocode lib

### PR DESCRIPTION
**Pull request by authorjapps**
### What to check in the PR?
+ Please check if anything else can be excluded
   + Please compare with the core `<artifactId>zerocode-tdd</artifactId>`
+ Please check any redundant exclusions made
    + i.e. something need not be excluded, but excluded

### How to validate?
+ Whether, we can add more exclusions to the list and see if the project is compiling as well as running fine(not breaking anything)
   + Issue this command `mvn clean install` by changing dir to `zerocode/junit5-testing`
+ Then we can take help of `mvn dependency:analyze-report` to check if anything is missing.
   + Sometimes this report is misleading, but gives a good suggession.

### CODE REVIEW

File: junit5-testing/pom.xml [hunk 1]
```diff
@@ -24,6 +24,127 @@
...
+                    <groupId>aopalliance</groupId>
+                    <artifactId>aopalliance</artifactId>
+                </exclusion>
+                -->
+                <exclusion>
+                    <groupId>org.jboss.resteasy</groupId>
+                    <artifactId>resteasy-jaxrs</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>javax.activation</groupId>
```
Comment (officiallysameer) [inline junit5-testing/pom.xml]: Could not find this in the parent pom

File: junit5-testing/pom.xml [hunk 2]
```diff
@@ -24,6 +24,127 @@
...
+                <exclusion>
+                    <groupId>com.google.guava</groupId>
+                    <artifactId>guava</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>junit</groupId>
+                    <artifactId>junit</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>javax.annotation</groupId>
```
Comment (officiallysameer) [inline junit5-testing/pom.xml]: Not used in parent project pom

File: junit5-testing/pom.xml [hunk 3]
```diff
@@ -24,6 +24,127 @@
...
+                    <artifactId>jsr250-api</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>commons-httpclient</groupId>
+                    <artifactId>commons-httpclient</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>aopalliance</groupId>
```
Comment (officiallysameer) [inline junit5-testing/pom.xml]: Not used in parent project pom

File: junit5-testing/pom.xml (hunk 1 above)
Comment (authorjapps) [reply junit5-testing/pom.xml]: Fixed.

File: junit5-testing/pom.xml (hunk 2 above)
Comment (authorjapps) [reply junit5-testing/pom.xml]: Fixed.

File: junit5-testing/pom.xml (hunk 3 above)
Comment (authorjapps) [reply junit5-testing/pom.xml]: Fixed

**Pull request merged**